src/main/java/com/commercetools/sync/commons/helpers/BaseSyncStatistics.java

```diff
@@ -0,0 +1,237 @@
+package com.commercetools.sync.commons.helpers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.netty.util.internal.StringUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Nonnull;
+import java.util.concurrent.TimeUnit;
+
+import static java.lang.String.format;
+
+public class BaseSyncStatistics {
+    private static final Logger LOGGER = LoggerFactory.getLogger(BaseSyncStatistics.class);
+
+    private String reportMessage;
+    private int updated;
+    private int created;
+    private int failed;
+    private int processed;
```

Comment:
Syncers should have ability to process resources in parallel, thus you should either document that class isn't thread safe (so that two threads don't depend on single instance of statistics - possible issue when you have one statistic instance in syncer, process resources in parallel and each processing thread try to increment counter at the very same moment) or adopt this to be thread safe (f.e. use AtomicInteger instead of int)